WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1601,6 +1601,16 @@ CreateContactGroupDialogTable   = @FullWidth@
 CreateContactGroupDialogCell    = padding:0 0 8px 0;
 CreateContactGroupDialogInput   = @FullWidth@
 
+#######################################################################
+#   Grant Rights Dialog
+#######################################################################
+
+GrantRightsDialogOuterContainer     = background-color:@lighten(SecondaryBrandingColor,94)@;
```

Comment:
The background color for different modals are different and are not consistent.
Say Attach Dialog has different background-color than  create new task folder, new mail folder,  shortcut, offline modal etc. 
And, it would not be a good idea to keep a generic background color for all the modal.
Attach Dialog and grant right dialog have very much different context in use.
Therefore, have used different property for GrantRightsDialogOuterContainer.